index.md

```diff
@@ -14,6 +14,10 @@ title: Clever
 
 [github.com/Clever/saml2](https://github.com/Clever/saml2)
 
+### Terms and Policies
+
+Clever's Terms and Policies are open source making it easy for people to view changes and historical information [github.com/Clever/policies](https://github.com/Clever/policies)
```

Comment:
I like that you added a little context here. It would be cool to do so for other projects as well.

--

One option to keep it always up to date.. we could fetch the repo descriptions via Github API and use that as the source of truth

<img width="1070" alt="Clever_saml2__Node_module_to_abstract_away_the_complexities_of_the_SAML_protocol_behind_an_easy_to_use_interface_" src="https://user-images.githubusercontent.com/102242/81430318-12dbf580-9114-11ea-9445-d07435d598d7.png">

Reply:
That's a great idea! I'll pull the descriptions from the repos and hard-code them for now. We can investigate adding an API integration as a followup later on 🙂